Spotify Stream Widget/LocalSongManager.vb

```diff
@@ -0,0 +1,111 @@
+﻿Imports Spotify_Stream_Widget.Logger
+
+Public Class LocalSongManager
+    'The purpose of this class is to give cover images for local songs because they aren't given by Spotify.
+    'To Use: Initialize with a directory path containing the local songs that will be played on Spotify
+    '( eg C:/Users/user/Music/ )
+    'Then call ImageForSong(title, artist) to get the album cover if there's a matching local song. 
+
+    Dim _dict As New Dictionary(Of String, String)
+    Dim _validExtensions As HashSet(Of String) = New HashSet(Of String)({".aa", ".aax", ".aac", ".aiff", ".ape", ".dsf", ".flac", ".m4a", ".m4b", ".m4p", ".mp3", ".mpc", ".mpp", ".ogg", ".oga", ".wav", ".wma", ".wv", ".webm"})
+    '   extensions from taglib docs: https://github.com/mono/taglib-sharp
+
+    Public Sub New(dir As String)
+        InitLocalDir(dir)
+    End Sub
+
+    Public Sub InitLocalDir(dir As String)
+        'Given a directory path, set list of local song files (will searched thru later by Viewer)
+        'Should be async, but too much of a hassle. 
+        'Only UX annoyance will be that thumbnail won't update live when local dir is set.
+
+        For Each filepath In IO.Directory.GetFiles(dir, "*.*").Where(Function(f) IsAudio(f))
```

Comment:
Does not work recursive. `System.IO.SearchOption.AllDirectories should fix` this.